extension/src/popup/router/pages/popup.vue

```diff
@@ -8,7 +8,7 @@
 
         <div>
             <div>
-                <h1>Your Extension</h1>
+                <h1>Your Watch With Me Extension</h1>
```

Comment:
Don't know if it's really useful.

```suggestion
                <h1>Watch With Me Extension</h1>
```